### DISCUSSION TITLE
How to measure pressure？

### DISCUSSION THREAD

**Discussion by duanlinlim** [InteractiveComputerGraphics/SPlisHSPlasH#340] [Q&A]
How to measure the pressure at a certain point in a dam failure case？

**Comment by janbender** [comment]
Some pressure solvers (e.g. WCSPH) compute the pressure for all partices, some others do not directly compute a pressure value. For these methods you would have to implement something to get this info. 

Finally, you can export the pressure data by using the scene setting "particleAttributes ": 

https://splishsplash.readthedocs.io/en/latest/file_format.html#export

**Comment by duanlinlim** [comment]
感谢上述回答，
我在场景文件中添加了	"enablePartioExport": true,和"particleAttributes": "velocity;density"但是只输出了 partio文件夹，没有输出particle文件夹

**Comment by janbender** [comment]
Can you please write in English?

**Comment by duanlinlim** [comment]
Thank you for the above answer,
I added "enablePartioExport": true and "particleAttributes": "velocity; density" in the scene file, but only output the particle folder, not the particle folder   
Can this method output the pressure at a specific location？

**Comment by duanlinlim** [comment]
I found in the GridModel_Sender2019.json file that "particoAttribles": "velocity" was added instead of "particleAttributes": "velocity", but no matter which one it is, there is no output from the particle folder

**Comment by duanlinlim** [comment]
Why did I press the 'l' key in Hotkeys, but there was no response and the particle information was not printed out

**Reply by janbender** [reply]
The hot key is "i".

**Comment by janbender** [comment]
If you enableVTKExport, particle files are exported which can be imported in ParaView. They contain all fields which are specified by "particoAttribles". To see which fields are available you can take a look at the tab "FluidModel" and see which options you have in the box "Color field".

**Comment by duanlinlim** [comment]
I would like to ask if it is necessary to select particles when printing particle information. Is it too blurry to circle particles directly on the GUI interface with the mouse,

**Comment by duanlinlim** [comment]
I would like to ask if it is necessary to select particles when printing particle information. Is it too blurry to circle particles directly on the GUI interface with the mouse,
